### PR TITLE
COR-1488-remove-searchbar

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

* 
* remove the search bar from the topical page (and topical only)
* The search is still available on GM-level pages (sidebar and GM selection page
* 

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-22 at 12 10 54

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-22 at 12 10 59

Screenshot 2023-03-22 at 12 14 47

Screenshot 2023-03-22 at 12 14 54

</details>

### CODE REVIEW

File: packages/app/src/domain/topical/components/search/hit-list.tsx [hunk 1]
```diff
@@ -1,151 +0,0 @@
...
-import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
-import { useSearchContext } from './context';
-
-interface HitListProps {
-  scope: 'vr' | 'gm';
-}
-
-export function HitList({ scope }: HitListProps) {
-  const { gmHits, vrHits, term, getOptionProps } = useSearchContext();
-  const { commonTexts } = useIntl();
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/topical/components/search/hit-list.tsx:19]: I think all these Sanity keys can be removed?

* `commonTexts.common.vr_plural`
* `commonTexts.common.vr_singular`
* `commonTexts.common.gm_plural`
* `commonTexts.common.gm_singular`
* `commonTexts.search.no_hits`

File: packages/app/src/domain/topical/components/search/search-input.tsx [hunk 2]
```diff
@@ -1,124 +0,0 @@
...
-import { VisuallyHidden } from '~/components/visually-hidden';
-import { useIntl } from '~/intl';
-import { fontSizes, space } from '~/style/theme';
-import { useSearchContext } from './context';
-
-const ICON_SPACE = 50;
-const ICON_SPACE_LARGE = 66;
-
-export function SearchInput() {
-  const { commonTexts } = useIntl();
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/topical/components/search/search-input.tsx:15]: I think all these Sanity keys can be removed?

* `commonTexts.search.clear`
* `commonTexts.search.placeholder`

File: packages/app/src/utils/use-scoped-warning.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { gmData, vrData } from '@corona-dashboard/common';
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/utils/use-scoped-warning.ts:1]: How exactly does this relate to this pull request?

**Review by VWSCoronaDashboard26 — COMMENTED** [review]
I see that this pull request also includes the removal of the VR combobox. While that is something that needs to be done regardless, I don't see the relationship between the homepage's search and the combobox for VR pages.

File: packages/app/src/domain/topical/components/search/search-input.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/topical/components/search/search-input.tsx:15]: You are correct, I removed them.

Please resolve this accordingly.

File: packages/app/src/domain/topical/components/search/hit-list.tsx (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/topical/components/search/hit-list.tsx:19]: You are correct, I removed them.

Please resolve this accordingly.

File: packages/app/src/domain/topical/components/search/context.tsx [hunk 4]
```diff
@@ -1,191 +0,0 @@
...
-      'aria-controls': id,
-      'aria-activedescendant': getOptionId(focusIndex),
-    },
-
-    comboboxProps: {
-      role: 'combobox',
-      'aria-expanded': showResults ? 'true' : 'false',
-      'aria-haspopup': 'grid',
-      'aria-owns': id,
-      'aria-label': commonTexts.search.aria_label,
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/topical/components/search/context.tsx:173]: You are correct, I removed them.

Please resolve this accordingly.

File: packages/app/src/domain/layout/components/vr-combo-box.tsx [hunk 5]
```diff
@@ -1,27 +0,0 @@
...
-export function VrComboBox(props: VrComboBoxProps) {
-  const { getLink, selectedVrCode } = props;
-
-  const { commonTexts } = useIntl();
-  const reverseRouter = useReverseRouter();
-  const router = useRouter();
-
-  return (
-    <ComboBox
-      placeholder={commonTexts.common.zoekveld_placeholder_regio}
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/layout/components/vr-combo-box.tsx:21]: You are correct, I removed them.

Please resolve this accordingly.

**Review by VWSCoronaDashboard26 — APPROVED** [review]
As previously discussed, I would prefer to not see history overwritten with commits.

File: packages/app/src/components/combo-box/combo-box.tsx [hunk 6]
```diff
@@ -33,9 +33,9 @@ type TProps<Option extends TOption> = {
  * ComboBox accept a generic type which extends `TOption` ({name: string}).
  *
  * ```ts
- * <Combobox<TVr> // generic passed here
+ * <Combobox<Gm> // generic passed here
  *   handleSelect={handleSafeRegionSelect}
- *   options={vr_collection
+ *   options={gm_collection
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/components/combo-box/combo-box.tsx:38]: Small questions, should this be `{gm_collection}`?

File: packages/app/src/pages/index.tsx [hunk 7]
```diff
@@ -225,14 +224,6 @@ const Home = (props: StaticProps<typeof getStaticProps>) => {
 
         <Spacer marginBottom={space[5]} />
 
-        <Box width="100%" backgroundColor="gray1" paddingY={space[5]}>
-          <Box paddingY={space[4]} paddingX={{ _: space[3], sm: space[4] }}>
-            <Search title={textShared.secties.search.title.nl} />
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/pages/index.tsx:230]: Is this value used on other places? If not, you might delete this key.

Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/index.tsx:230]: Indeed it is not used elsewhere. I adjusted it, please resolve it accordingly.

**Review by VWSCoronaDashboard21 — APPROVED** [review]

**Pull request merged**